assets/index.pug

```diff
@@ -54,5 +54,5 @@ head
 // script(src='https://cdn.jsdelivr.net/npm/lodash@4.17.15')
 // script(src='https://cdn.jsdelivr.net/npm/toastify-js@1.7.0')
 // script(src='https://cdn.jsdelivr.net/npm/@tarekraafat/autocomplete.js@7.2.0/dist/js/autoComplete.min.js')
-script(src='../dist/app.js' type='module')
+script(src='./app.js' type='module')
```

Comment:
Things still seem to work after running `yarn bundle`, but adding this also allows for hot-reloading when using `yarn dev` (but maybe there was already a way of doing that with existing setup)